test/unit/auth/auth.spec.ts

```diff
@@ -501,6 +501,38 @@ AUTH_CONFIGS.forEach((testConfig) => {
           });        
       });
 
+      it('verifyIdToken() should reject user disabled before ID tokens revoked', () => {
+        const expectedAccountInfoResponse = getValidGetAccountInfoResponse(tenantId);
+        const expectedAccountInfoResponseUserDisabled = Object.assign({}, expectedAccountInfoResponse);
+        expectedAccountInfoResponseUserDisabled.users[0].disabled = true;
+        const expectedUserRecordDisabled = getValidUserRecord(expectedAccountInfoResponseUserDisabled);
+        const validSince = new Date(expectedUserRecordDisabled.tokensValidAfterTime!);
+        // One second before validSince.
+        const oneSecBeforeValidSince = Math.floor(validSince.getTime() / 1000 - 1);
+        const getUserStub = sinon.stub(testConfig.Auth.prototype, 'getUser')
+          .resolves(expectedUserRecordDisabled);
+        expect(expectedUserRecordDisabled.disabled).to.be.equal(true);
+        stubs.push(getUserStub);
+
+        const unsignedToken = mocks.generateIdToken({
+          algorithm: 'none',
```

Comment:
Can we not depend on algorithm being none which is specific to auth emulator.
You can just follow existing tests that touch on revocation:
https://github.com/firebase/firebase-admin-node/blob/a9d8b410848babfa9b1dfeedbf1ae00572deea57/test/unit/auth/auth.spec.ts#L541